receiver/awsecscontainermetricsreceiver/testdata/task_metadata.json

```diff
@@ -0,0 +1,106 @@
+{
```

Comment:
Is/can this test data being used anywhere in these changes or existing structure?  Not immediately clear why it's being introduced.

Reply:
Actually, it was a big PR. Later we decided to to split it into 3 PRs to make the review process faster. In this PR we are mainly adding the structs and test data. This is being used by our code which are coming with next PR. 
Yea, we can debate but I think it's fine.